clang/include/clang/AST/TypeBase.h

```diff
@@ -2956,6 +2956,9 @@ class alignas(TypeAlignment) Type : public ExtQualsTypeCommonBase {
     return TST;
   }
 
+  const TemplateSpecializationType *
```

Comment:
Can we also have a documentation comment explaining what the function does and what is the difference from `getAsNonAliasTemplateSpecializationType` ?